core/object.h

```diff
@@ -335,6 +335,11 @@ protected:
 			m_inherits::_notificationv(p_notification, p_reversed);                                                                     \
 	}                                                                                                                                   \
                                                                                                                                         \
+protected:                                                                                                                              \
+	virtual void _destruct() {                                                                                                \
```

Comment:
Not sure why as I can't see the logs proper now, but clang-format seems unhappy with this line.